gnovm/tests/files/zrealm3.gno

```diff
@@ -57,18 +57,37 @@ func main() {
 //         }
 //     ],
 //     "ObjectInfo": {
-//         "ID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:5",
+//         "ID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:7",
+//         "ModTime": "0",
+//         "OwnerID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:6",
+//         "RefCount": "1"
+//     }
+// }
+// c[a8ada09dee16d791fd406d629fe29bb0ed084a30:6]={
+//     "ObjectInfo": {
+//         "ID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:6",
 //         "ModTime": "0",
 //         "OwnerID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:2",
 //         "RefCount": "1"
+//     },
+//     "Value": {
+//         "T": {
+//             "@type": "/gno.RefType",
+//             "ID": "gno.land/r/test.Node"
+//         },
+//         "V": {
+//             "@type": "/gno.RefValue",
+//             "Hash": "567a18d9c7594ece7956ce54384b0858888bb834",
+//             "ObjectID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:7"
+//         }
```

Comment:
You can tell it's a HeapItemValue because it has one field, "Value".
(we could update the tests to show the type in the future for clarity,
like `c[a8ada09dee16d791fd406d629fe29bb0ed084a30:6]=*HeapItemValue{`

Reply:
While this could be eliminated with some intelligence prior to persistence,
to unwrap the HeapItemValue in case where the HeapItemValue only has refcount of 1,
1. The HeapItemValue is necessary when the .Value is a primitive non-object anyways, and
2. This would mean PointerValue.Base is nil, and we'd need additional logic to re-wrap when necessary.
3. And with the above point, it's not clear the result would be any faster.
But this is something we could explore after launch.